packages/driver/cypress/integration/commands/request_spec.js

```diff
@@ -1316,6 +1316,20 @@ describe('src/cy/commands/request', () => {
 
           cy.request({ url: 'http://localhost:1234/foo', timeout: 50 })
         })
+
+        // @see https://github.com/cypress-io/cypress/issues/15101
+        it('can receive timeout errors from the backend', (done) => {
```

Comment:
I tried this test out on `develop` and it passes, so while it seems like a legit test, I'm not sure it tests this issue.

Reply:
Good call, this repro actually was working reliably for me at the start but... I guess something changed at some point, this no longer reliably reproduces the issue.

I've noticed the 2 user-supplied repos are flaky to fail too: https://github.com/iondrimba/cypress_maximum_call_stack https://github.com/TimVee/deconstructPacked-bug

I reverted this test and created a @packages/socket test instead to reliably reproduce the issue.